SurveyTests/Presentation/Modules/Login/LoginViewSpec.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  LoginViewSpec.swift
+//  SurveyTests
+//
+//  Created by Khanh on 08/08/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Combine
+import Nimble
+import Quick
+
+@testable import Survey
+
+final class LoginViewSpec: QuickSpec {
+
+    override func spec() {
+        var loginView: LoginView!
+
+        describe("a LoginView") {
+            beforeEach {
+                loginView = LoginView(viewModel: LoginViewModel(loginUseCase: LoginUseCaseMock(),
```

Comment:
I think this line break should be
```suggestion
                loginView = LoginView(
                    viewModel: LoginViewModel(
                        loginUseCase: LoginUseCaseMock(),
                        storeUseCase: StoreTokenUseCaseMock()
                    )
                )
```